docs/pages/management/operations/proxy-peering.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Proxy Peering Migration
+description: How to upgrade an existing Teleport cluster to proxy peering mode.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="10.0"
+>
+  TLS routing is available starting from Teleport `10.0`.
+</Details>
+
+## Requirements
+
+Teleport Proxy servers must be able to reach each other over the network. Ensure
+there are no firewall policies that would block proxy to proxy communication.
```

Comment:
How would a reader know whether there are firewall policies that would block communication between Proxy Service instances?

Reply:
What to look for specifically? Some policy that would not allow inbound connections on port 3021 (default port for proxy peering). When Proxy Peering is not enabled there is no communication between Proxy instances so someone with an existing cluster may be blocking this traffic by default.

Reply:
Specified port here [2224ced](https://github.com/gravitational/teleport/pull/15790/commits/2224ced7bb3279a5786cb51463b943710580bd03)